cmd/destroy.go

```diff
@@ -49,7 +49,8 @@ func init() {
 	destroyCmd.Flags().BoolVarP(&common.Graceful, "graceful", "", false,
 		"attempt to stop containers before removing")
 	destroyCmd.Flags().BoolVarP(&all, "all", "a", false, "destroy all containerlab labs")
-	destroyCmd.Flags().BoolVarP(&yes, "yes", "y", false, "auto-approve deletion when used with --all (skips confirmation prompt)")
+	destroyCmd.Flags().BoolVarP(&yes, "yes", "y", false,
```

Comment:
some of this was just gofumpt + the line fixer thing, some I just broke out args onto line per arg where I saw some long ass lines